src/javascripts/components/tabs.js

```diff
@@ -33,45 +48,53 @@
       $(id).show().removeAttr('aria-hidden')
     },
 
-    // Accordion mode
-    clickAccordionItem: function (e) {
+    // Tabs mode
+    clickTabItem: function (e) {
       e.preventDefault()
 
+      // Get tab container ID
       var id = $(this).attr('href')
 
-      // Let the main tabs deal with the states
-      $('.app-c-tabs__item a[href="' + id + '"]').click()
+      // Activate current tab
+      GOVUK.tabs.activateCurrentTab(id)
     },
 
     // Close current container on click
     clickCloseContainer: function (e) {
       e.preventDefault()
-      $('.app-c-tabs__item').removeClass('app-c-tabs__item--current')
-      $('.app-c-tabs__heading').removeClass('app-c-tabs__heading--current')
-      $('.app-c-tabs__container').hide().attr('aria-selected', 'true')
+
+      // Get tab container ID and save example object
+      var id = $(this).parent().attr('id')
+
+      // Reset tabs
+      GOVUK.tabs.resetTabs('#' + id)
     },
 
-    init: function () {
+    init: function (selector) {
       // If more than one container
-      if ($('.app-c-tabs__container').length > 1) {
-        // Hide all containers
-        $('.app-c-tabs__container').hide()
-
-        // Add close button to each container
-        $('.app-c-tabs__container').append('<a class="app-c-link--close app-o-chevron--top" href="#close">Close</a>')
-      }
+      var $examples = $(selector)
+      $.each($examples, function (key, obj) {
+        if ($(obj).find('.js-tabs__container').length > 1) {
+          // Hide all containers
+          $(obj).find('.js-tabs__container').hide()
+
+          // Add close button to each container
+          $(obj).find('.js-tabs__container').append('<a class="app-c-link--close js-link--close app-o-chevron--top" href="#close">Close</a>')
```

Comment:
This might make more sense to be a button, raised an issue since there's other things like this. https://github.com/alphagov/govuk-design-system/issues/124